README.md

```diff
@@ -104,7 +104,8 @@ After you're finished please remove all the comments and instructions!
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](#)
+- [Live Demo Link](https://smart-budget-aw9r.onrender.com/)
+- [Loom video link](https://www.loom.com/share/01c6a54f42404e23b97204d00e8e3ad0)
```

Comment:
- [ ] **You have done a great job completing the project requirements 👏 👏**. However, the link to your loom video is not available in your README, and the live demo link provided is pointing to the repository itself instead of opening the corresponding links 😨 

   Kindly fix this small issue to make your project more professional 👍 

   ![image](https://github.com/techEmprez/SmartBudget/assets/102757126/27647e24-229b-4a20-b0e0-1e6ed54af7b5)

Reply:
Pleas, can you refresh  as all changes were adequately made and is present from my end.

Please reviewer, kindly reach out for clarification 

Please, Thank you.